Client/Frontend/Strings.swift

```diff
@@ -797,3 +797,10 @@ extension Strings {
 extension String {
     public static let FxAQRCode_Instructions = NSLocalizedString("fxa.qr-scanning-view.instructions", value: "Scan the QR code shown at firefox.com/pair", comment: "Instructions shown on qr code scanning view")
 }
+
+//Today Widget Strings - New search / New Private Search
+extension Strings {
```

Comment:
@noorhashem This is the only part we need when we land strings. Rest you can do in your refactored PR as then it'll make the PR very clean and easy to revert / track for string changes